releasing.md

```diff
@@ -155,14 +155,46 @@ ks apply ${ENV} -c workflows
 Create a PR to update [jupyterhub_spawner.py](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/jupyterhub_spawner.py#L15) 
 to point to the newly built Jupyter notebook images.
 
-## Update the ksonnet configs
+## Create a release branch (if necessary)
 
-Create a PR to update the ksonnet configs to use the latest images built in the previous steps.
+If you aren't already working on a release branch (of the form `v${VER}-branch`, where `${VER}` is a major-minor version number), then create one.  Release branches serve several purposes:
```

Comment:
I would suggest dropping 'v${VER}'
branch seems redundant
The branch will be used with ksonnet so that users can pull a stable version of the configs rather than the latest configs on master.

Reply:
Git's shared namespace for different kinds of refs can lead to user confusion.  I'm suggesting `-branch` to disambiguate between moving targets (e.g., the branch from which all v0.1-series releases will be tagged) and fixed targets (e.g., the tags v0.1, v0.1.1, etc).  If you're making the argument that `v0.1` should be a branch but `v0.1.0` should be a tag, I can see that, but I think it makes sense to choose names that make the distinction more explicit to users (especially since this will be the name they give to ksonnet).

Reply:
Makes sense.